app/models/role.rb

```diff
@@ -8,4 +8,6 @@ class Role < ApplicationRecord
   has_many :users, through: :assignments
 
   validates :name, presence: true, uniqueness: true
```

Comment:
Since the name gets downcased now, it'd make sense to validate its uniqueness disregarding the case:

```ruby
validates :name, presence: true, uniqueness: { case_sensitive: false }
```